bind/gencpy.go

```diff
@@ -243,11 +243,17 @@ func (g *cpyGen) gen() error {
 
 	for _, v := range g.pkg.vars {
 		name := v.Name()
+		get := v.doc
+		set := v.doc
+		if v.doc == "" {
```

Comment:
@sbinet 
I agree with you. And this might be a not easy issue when we consider Python extension module structure.
How about just left `FIXME` comment.